jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettings.kt

```diff
@@ -24,12 +24,19 @@ class AwsSettings : PersistentStateComponent<AwsConfiguration> {
             state.isTelemetryEnabled = value
         }
 
+    var toolkitVersion: String?
```

Comment:
Can we have a configuration explicitly named around "first time use"? ToolkitVersion is open ended

Reply:
👍 or even more specific "promptedForTelemetry"

Reply:
Done.